docs/integration-guides/websockets.md

```diff
@@ -0,0 +1,504 @@
+
+```json
+{
+  "action": "subscribe",
+  "topic": "confirmation"
+}
+```
+
+Unsubscribing also has the format:
+
+To unsubscribe:
+```json
+{
+  "action": "unsubscribe",
+  "topic": "confirmation"
+}
+```
+
+**Optional Filters**
+
+Some topics support filters as well. Details of the subscription filter options for each topic are included in examples below.
+
+!!! note
+    Note that, if **empty** `options` are supplied (see examples below), an empty filter will be used and nothing will be broadcasted.
+
+**Available Topics**
+
+Current topics available for subscribing to include:
+
+* `confirmation`
+* `vote` (experimental, unfinished)
+* `stopped_election`
+* `active_difficulty`
+
+---
+
+#### Confirmations
+
+--8<-- "multiple-confirmation-notifications.md"
+
+##### Subscribing
+
+To subscribe to all confirmed blocks:
+
+```json
+{
+  "action": "subscribe",
+  "topic": "confirmation"
+}
+```
+
+##### Filtering options
+
+###### Confirmation types
+
+The node classifies block confirmations into the following categories:
+
+* **Active quorum**: a block is confirmed through voting (including `block_confirm` RPC if block is previously unconfirmed)
+* **Active confirmation height**: a block which is confirmed as a dependent election from a successor through voting (or by `block_confirm` RPC if the block is already confirmed)
+* **Inactive**: a block that is not in active elections is implicitly confirmed by a successor.
+
+By default, the node emits **all** confirmations to WebSocket clients. However, the following filtering option is available:
+
+```json
+{
+  "action": "subscribe",
+  "topic": "confirmation",
+  "options": {
+    "confirmation_type": "<type>"
+  }
+}
+```
+
+The most common values for `confirmation_type` are `all` (default), `active` and `inactive`.
+
+If more fine-grained filtering is needed, `active` can be replaced with `active_quorum` or `active_confirmation_height` per the definitions above.
+
+###### Accounts
+
+Filters for **confirmation** can be used to subscribe only to selected accounts. Once filters are given, blocks from accounts that do not match the options are not broadcasted.
+
+!!! warning "Legacy blocks never broadcasted"
+    Note that [legacy blocks](/glossary#legacy-blocks) are never broadcasted if filters are given, even if they match the accounts.
+
+```json
+{
+  "action": "subscribe",
+  "topic": "confirmation",
+  "options": {
+    "all_local_accounts": true,
+    "accounts": [
+      "nano_16c4ush661bbn2hxc6iqrunwoyqt95in4hmw6uw7tk37yfyi77s7dyxaw8ce",
+      "nano_3dmtrrws3pocycmbqwawk6xs7446qxa36fcncush4s1pejk16ksbmakis32c"
+    ]
+  }
+}
+```
+
+* When `all_local_accounts` is set to **`true`**, blocks that mention accounts in any wallet will be broadcasted.
+* `accounts` is a list of additional accounts to subscribe to. Both prefixes are supported.
+
+??? tip "Updating the list of accounts"
```

Comment:
You mentioned wanting to expand this by default, is that still the case?

Reply:
Yeah, updated